components/multi-platform-controller/production/stone-prd-rh01/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
just for my information - is there any particular reason some servers starts from `stone-*-*` and some are `kflux-*-*` or it is just historical heritage?